packages/usa-alert/src/styles/_usa-alert.scss

```diff
@@ -2,18 +2,13 @@
 
 @use "uswds-core" as *;
 
-// Alert variables ---------- //
-$alert-icons: (
-  success: "check_circle",
-  warning: "warning",
-  error: "error",
-  info: "info",
-  emergency: "error",
-);
-
 .usa-alert {
   @include alert-styles;
 
+  .usa-alert__body {
+    @include u-margin-x($theme-alert-padding-x);
```

Comment:
Yes, good note. I updated the alignment to use padding instead of margin in [069cb7b](https://github.com/uswds/uswds/pull/5636/commits/069cb7b745ebdffa0cbd2fc744ba698061d1c766) and [914ca8a](https://github.com/uswds/uswds/pull/5636/commits/914ca8abfc332bd4655b4a09510caef531bd3055)